{
	"id": "SUSE-SU-2024:4159-1",
	"modified": "2024-12-03T15:30:11Z",
	"published": "2024-12-03T15:30:11Z",
	"related": [
		"CVE-2024-11168"
	],
	"summary": "Security update for python3",
	"details": "This update for python3 fixes the following issues:\n\n- CVE-2024-11168: Fixed improper validation of IPv6 and IPvFuture addresses (bsc#1233307)\n\nOther fixes:\n- Remove -IVendor/ from python-config (bsc#1231795)\n",
	"affected": [
		{
			"package": {
				"ecosystem": "SUSE:Linux Enterprise Micro 5.1",
				"name": "python3",
				"purl": "pkg:rpm/suse/python3&distro=SUSE%20Linux%20Enterprise%20Micro%205.1"
			},
			"ranges": [
				{
					"type": "ECOSYSTEM",
					"events": [
						{
							"introduced": "0"
						},
						{
							"fixed": "3.6.15-150000.3.167.1"
						}
					]
				}
			],
			"ecosystem_specific": {
				"binaries": [
					{
						"libpython3_6m1_0": "3.6.15-150000.3.167.1",
						"python3": "3.6.15-150000.3.167.1",
						"python3-base": "3.6.15-150000.3.167.1"
					}
				]
			}
		},
		{
			"package": {
				"ecosystem": "SUSE:Linux Enterprise Micro 5.1",
				"name": "python3-core",
				"purl": "pkg:rpm/suse/python3-core&distro=SUSE%20Linux%20Enterprise%20Micro%205.1"
			},
			"ranges": [
				{
					"type": "ECOSYSTEM",
					"events": [
						{
							"introduced": "0"
						},
						{
							"fixed": "3.6.15-150000.3.167.1"
						}
					]
				}
			],
			"ecosystem_specific": {
				"binaries": [
					{
						"libpython3_6m1_0": "3.6.15-150000.3.167.1",
						"python3": "3.6.15-150000.3.167.1",
						"python3-base": "3.6.15-150000.3.167.1"
					}
				]
			}
		}
	],
	"references": [
		{
			"type": "ADVISORY",
			"url": "https://www.suse.com/support/update/announcement/2024/suse-su-20244159-1/"
		},
		{
			"type": "REPORT",
			"url": "https://bugzilla.suse.com/1231795"
		},
		{
			"type": "REPORT",
			"url": "https://bugzilla.suse.com/1233307"
		},
		{
			"type": "WEB",
			"url": "https://www.suse.com/security/cve/CVE-2024-11168"
		}
	]
}
